youtube_dl/extractor/qqmusic.py

```diff
@@ -352,7 +357,7 @@ def _real_extract(self, url):
         cdlist = list_json['cdlist'][0]
         entries = [
             self.url_result(
-                'http://y.qq.com/#type=song&mid=' + song['songmid'], 'QQMusic', song['songmid']
+                'https://y.qq.com/n/yqq/song/' + song['songmid'] + ".html", 'QQMusic', song['songmid']
```

Comment:
In general we use single quotes unless there's a good reason. (single quotes in strings, etc.)